ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -123,12 +249,51 @@ export class DkgCreateCommand extends IronfishCommand {
     return result
   }
 
+  async performRound1WithLedger(
+    ledger: Ledger,
+    client: RpcClient,
+    participantName: string,
+    identities: string[],
+    minSigners: number,
+  ): Promise<{
+    round1: { secretPackage: string; publicPackage: string }
+  }> {
+    const identityResponse = await client.wallet.multisig.getIdentity({ name: participantName })
+    const identity = identityResponse.content.identity
+
+    if (!identities.includes(identity)) {
+      identities.push(identity)
+    }
+
+    // TODO(hughy): determine how to handle multiple identities using index
+    const { publicPackage, secretPackage } = await ledger.dkgRound1(0, identities, minSigners)
+
+    this.log('\nRound 1 Encrypted Secret Package:\n')
+    this.log(secretPackage.toString('hex'))
+    this.log()
+
+    this.log('\nRound 1 Public Package:\n')
+    this.log(publicPackage.toString('hex'))
+    this.log()
+
+    this.log('Next step:')
+    this.log('Send the round 1 public package to each participant')
```

Comment:
this causes double printing of these messages

Reply:
removed